### PR TITLE
Add current index to each block

### PR DESCRIPTION
**Pull request by jdelight**

### CODE REVIEW

File: tasks/collate.js [hunk 1]
```diff
@@ -68,13 +68,17 @@ var registerHelper = function (item) {
...
- *   <li>List Item</li>
+ *   <li>List Item {{@index}}</li>
  * {{/iterate}}
  */
 Handlebars.registerHelper('iterate', function (n, block) {
-	var accum = '';
+	var accum = '', data;
 	for (var i = 0; i < n; ++i) {
-		accum += block.fn(i);
+		if (block.data){
```
Comment (LukeAskew) [inline tasks/collate.js]: One small nitpick...add a space before the opening paren `if (block.data) {`

Comment (jdelight) [inline tasks/collate.js]: Good spot :+1:

**Comment by LukeAskew** [comment]
:palm_tree:

**Comment by jdelight** [comment]
:dancer: Thanks!

**Pull request merged**